docs/CMock_Summary.md

```diff
@@ -518,6 +518,14 @@ from the defaults. We've tried to specify what the defaults are below.
   * `:include` will mock externed functions
   * `:exclude` will ignore externed functions (default).
 
+* `:treat_inlines`:
```

Comment:
Thank you so much for remembering to add this to the documentation!

Reply:
No problem, thanks for the review!